kombu/transport/redis_cluster.py

```diff
@@ -341,18 +341,19 @@ def _brpop_read(self, **options):
             resp = self.parse_response(conn, 'BRPOP', **options)
         except self.connection_errors:
             raise Empty()
-        except MovedError:
-            raise Empty()
         conn.client.connection.send_command('BRPOP', conn.key, conn.timeout) # schedule next BRPOP
 
         if resp:
             self.deliver_response(resp)
             return True
 
     def _poll_error(self, cmd, conn, **options):
-        resp = self.parse_response(conn, 'BRPOP', **options)
-        if resp:
-            self.deliver_response(resp)
+        try:
+            resp = self.parse_response(conn, 'BRPOP', **options)
+            if resp:
+                self.deliver_response(resp)
+        except self.connection_errors:
+            pass
```

Comment:
any reason for just passing?

Reply:
We should not throw error on `_poll_error` method to continue kombu to work.

Reply:
Ok then let's log exception at least

Reply:
and please add comments why we should not throw an error on that method